### PR TITLE
Initial version of the NMaaS presentations page

### LINKED ISSUE #33: Add a page about past NMaaS presentations
A page which would aggerate past NMaaS presentations given on conferences/events should be created on docs.nmaas.eu.

### PR DESCRIPTION
**Pull request by korvoj**
Currently work in progress, not to be merged.

### CODE REVIEW

File: docs/nmaas-presentations.md [hunk 1]
```diff
@@ -0,0 +1,506 @@
...
+
+**Event:** 4th SIG-NOC Meeting
+
+**Event Homepage:** [https://wiki.geant.org/display/SIGNOC/4th+SIG-NOC+meeting](https://wiki.geant.org/display/SIGNOC/4th+SIG-NOC+meeting)
+
+**Date:** 28-29 November 2016
+
+**Location:** Cambridge, United Kingdom
+
+**Slides:** Link to the presentation.
```
Comment (korvoj) [inline docs/nmaas-presentations.md]: ID: 1

File: docs/nmaas-presentations.md [hunk 2]
```diff
@@ -0,0 +1,506 @@
...
+
+**Event:** NORDUnet Technical Workshop 2017
+
+**Event Homepage:** [https://events.nordu.net/display/NTW2017/Programme](https://events.nordu.net/display/NTW2017/Programme)
+
+**Date:** 12-14 September 2017
+
+**Location:** Copenhagen, Denmark
+
+**Slides:** Link to the presentation.
```
Comment (korvoj) [inline docs/nmaas-presentations.md]: ID 3

File: docs/nmaas-presentations.md [hunk 3]
```diff
@@ -0,0 +1,506 @@
...
+
+**Presenter(s):** Jakub Gutkowski, Lukasz Lopatowski
+
+**Event:** GÉANT Project Symposium 2017
+
+**Date:** 3-4 October 2017
+
+**Location:** Budapest, Hungary
+
+**Slides:** Link to the presentation.
```
Comment (korvoj) [inline docs/nmaas-presentations.md]: ID 4

File: docs/nmaas-presentations.md [hunk 4]
```diff
@@ -0,0 +1,506 @@
...
+
+**Event:** SuperComputing 2017 (SC17)
+
+**Event Homepage:** [https://sc17.supercomputing.org/index.html](https://sc17.supercomputing.org/index.html)
+
+**Date:** 12-17 November 2017
+
+**Location:** Denver, USA
+
+**Slides:** Link to the presentation.
```
Comment (korvoj) [inline docs/nmaas-presentations.md]: ID 5

File: docs/nmaas-presentations.md [hunk 5]
```diff
@@ -0,0 +1,506 @@
...
+
+**Event:** 5th SIG-PMV
+
+**Event Homepage:** [https://wiki.geant.org/display/PMV/5th+SIG-PMV+Meeting+@+Manchester+UK](https://wiki.geant.org/display/PMV/5th+SIG-PMV+Meeting+@+Manchester+UK)
+
+**Date:** 23-24 October 2018
+
+**Location:** Manchester, UK
+
+**Slides:** Link to the presentation.
```
Comment (korvoj) [inline docs/nmaas-presentations.md]: ID 9

File: docs/nmaas-presentations.md [hunk 6]
```diff
@@ -0,0 +1,506 @@
...
+
+**Event:** SuperComputing 2018 (SC18)
+
+**Event Homepage:** [https://sc18.supercomputing.org/index.html](https://sc18.supercomputing.org/index.html)
+
+**Date:** 13-14 November 2018
+
+**Location:** Dallas, USA
+
+**Slides:** Link to the presentation.
```
Comment (korvoj) [inline docs/nmaas-presentations.md]: ID 10

File: docs/nmaas-presentations.md [hunk 7]
```diff
@@ -0,0 +1,506 @@
...
+
+**Event:** 6th SIG-PMV
+
+**Event Homepage:** [https://wiki.geant.org/display/PMV/6th+SIG-PMV+Meeting+@+Dublin](https://wiki.geant.org/display/PMV/6th+SIG-PMV+Meeting+@+Dublin)
+
+**Date:** 2-3 July 2019
+
+**Location:** Dublin, Ireland
+
+**Slides:** Link to the presentation.
```
Comment (korvoj) [inline docs/nmaas-presentations.md]: ID 11

File: docs/nmaas-presentations.md [hunk 8]
```diff
@@ -0,0 +1,506 @@
...
+
+**Event:** Workshop on Network Management and Monitoring
+
+**Event Homepage:** [https://wiki.geant.org/display/PUB/Workshop+on+Network+Management+and+Monitoring](https://wiki.geant.org/display/PUB/Workshop+on+Network+Management+and+Monitoring)
+
+**Date:** 21-22 October 2019
+
+**Location:** Copenhagen, Denmark
+
+**Slides:** Link to the presentation.
```
Comment (korvoj) [inline docs/nmaas-presentations.md]: ID 12

File: docs/nmaas-presentations.md [hunk 9]
```diff
@@ -0,0 +1,506 @@
...
+
+**Event:** 18th Service and Technology Forum (STF)
+
+**Event Homepage:** https://wiki.geant.org/display/APM/18th+STF+-+Copenhagen%2C+October+2019
+
+**Date:** 22-23 October 2019
+
+**Location:** Copenhagen, Denmark
+
+**Slides:** Link to the presentation.
```
Comment (korvoj) [inline docs/nmaas-presentations.md]: ID 13

File: docs/nmaas-presentations.md [hunk 10]
```diff
@@ -0,0 +1,506 @@
...
+
+**Presenter(s):** Lukasz Lopatowski, Vojdan Kjorveziroski
+
+**Event:** GÉANT Project Symposium 2020
+
+**Date:** 4-5 February 2020
+
+**Location:** Ljubljana, Slovenia
+
+**Slides:** Link to the presentation.
```
Comment (korvoj) [inline docs/nmaas-presentations.md]: ID 14

File: docs/nmaas-presentations.md [hunk 11]
```diff
@@ -0,0 +1,506 @@
...
+
+**Presenter(s):** Lukasz Lopatowski, Frederic Loui, Damiano Verzulli
+
+**Event:** GARR Workshop 2020
+
+**Date:** 9th November 2021
+
+**Location:** Online
+
+**Slides:** Link to the presentation.
```
Comment (korvoj) [inline docs/nmaas-presentations.md]: ID 18

**Comment by korvoj** [comment]
@llopat , I added the ID numbers of the presentations as used in the file naming strategy. Please remove the entries that you think should not be present in the list and add the missing links, if possible.

**Comment by llopat** [comment]
@korvoj thanks, I will complete this by noon

**Comment by llopat** [comment]
@korvoj I added the missing links and removed some of the older presentations from the list. Maybe you could go through the links and see if they work and point to correct files

**Comment by korvoj** [comment]
@llopat , thanks, the links look good. I am going to merge this.

**Pull request merged**